src/transaction.js

```diff
@@ -18,7 +18,9 @@
 
 var common = require('@google-cloud/common');
 var extend = require('extend');
+var grpc = require('grpc');
```

Comment:
`grpc` isn't our own dependency, so I don't think we should load it up through whichever other module brings it in.

Reply:
Doh, I hadn't even noticed that. I just added a commit to pull it from gax instead.